Gruntfile.coffee

```diff
@@ -957,6 +957,9 @@ module.exports = (grunt) ->
 						"The “contentinfo” role is unnecessary for element “footer”."
 						"The “navigation” role is unnecessary for element “nav”."
 						"The “banner” role is unnecessary for element “header”."
+						"Element “search” not allowed as child of element “main” in this context."
+						"Element “search” not allowed as child of element “section” in this context."
+						"Element “search” not allowed as child of element “div” in this context."
```

Comment:
Here's a fix for the ``htmllint:all`` task's failures :).

Needed an extra part at the end of the error messages.

```suggestion
						"Element “search” not allowed as child of element “main” in this context. (Suppressing further errors from this subtree.)"
						"Element “search” not allowed as child of element “section” in this context. (Suppressing further errors from this subtree.)"
						"Element “search” not allowed as child of element “div” in this context. (Suppressing further errors from this subtree.)"
```